google/cloud/aiplatform/training_jobs.py

```diff
@@ -1496,6 +1557,30 @@ def run(
                 Additionally the dataset format is passed in as:
 
                 os.environ["AIP_DATA_FORMAT"]
+            annotation_schema_uri (str):
+                Only applicable to custom training.
```

Comment:
This line can be removed because this argument should only be made available in the Custom Training classes.